client/src/tests/CardList.test.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import * as rtl from "@testing-library/react";
+import { CardList } from "../components/CardList";
+
+describe("players length", () => {
```

Comment:
wait I stand corrected you broke the tests up as well freaking great work sorry I doubted you